### ISSUE TITLE
Text of email messages

### ISSUE THREAD

**Issue by keithgee** [amandafrench/VT-DSpace#6]
Please someone review the texts of the emails in
 branch: dspace-5_x VT-DSpace/dspace/config/emails/ and change as necessary.

**Comment by alawvt** [comment]
Amanda,

I'll assume that you are handling this, since you said in the last meeting that you wanted to review the emails.

Thanks,

Anne

**Comment by amandafrench** [comment]
Ah, I knew I saw this issue somewhere! I just tested one such change, but I'm going to have to figure out this whole branch thing. We'll have to pick either GitHub or GitLab for issue tracking ... I'd prefer GitLab, I think. I'll recreate this issue there.

**Comment by keithgee** [comment]
Variety is the spice of life! I was trying to put issues that would be addressed in GitHub files in the GitHub tracker, and issues that would be addressed in the GitLab code on GitLab. Other issues are ignored.

I don't have the strength to lead a mutiny, so I will acquiesce.

**Comment by amandafrench** [comment]
No, no, I may not understand fully. Is it necessary to change config files here because they're in .gitignore?

**Comment by keithgee** [comment]
I've been thinking of this as two separate projects.

1. The Vagrant and Ansible project that Francis worked out to provision a virtual machine and install DSpace and all of its prerequisites. This is currently on the VT GitLab server.

2. The DSpace source code, with our own VTechWorks customizations. My memory is a little fuzzy here (I"ll have to look through some history), but I think we've tried this in several locations since we started working on this upgrade - in VT GitLab, on GitHub in your account as a fork of the main DSpace project (which is where I opened this issue), and now finally as a copy of that fork in the VTUL organization on GitHub.

Most of the systems related work is in project 1, and most of the changes to DSpace will be made in project 2. 

I might have misled above with the way I worded things so that it sounded like you have to modify the DSpace files from the GitHub web interface. That's not true - you can modify with any editor and then interact with github with the git command or any git interface.  I was just pointing out that some of the files we've been modifying for this upgrade are in GitLab and some are in GitHub.

Whenever I've been making a change to our version of the DSpace source code, my process has been to create a branch on my laptop for the change, make changes and commit to that branch as necessary, test in the LDE, and when the change is finished and working, push that branch up to GitHub. Then I do a pull request to pull the new feature branch into the 5_x branch of our project on GitHub. Ideally, someone else would review the code for the new feature and approve the pull request. I don't have clear, step-by-step documentation for how to do these things. I want to configure my machine to use the copy of the project in the VTUL organization in GitHub(is this the final location?), and then for us to walk through these steps together for understanding/troubleshooting/documentation.

Also, once everyone can see what this is, we can talk about whether that's an appropriate workflow or not for our group.

As for the issues list - I think you may be right that it's best to have all the issues in one place. After all, there will be some tasks that involve changes in both projects, and some issues that will be opened before we know which project needs to be modified. And it's just easier to look in one place. 

I was completely kidding about the mutiny. Best captain ever!
